packages/app/src/cli/models/app/app.ts

```diff
@@ -252,6 +271,32 @@ export class App implements AppInterface {
   }
 }
 
+export function validateFunctionExtensionsWithUiHandle(
+  functionExtensionsWithUiHandle: ExtensionInstance<FunctionConfigType>[],
+  allExtension: ExtensionInstance[],
+): string[] | undefined {
+  const errors: string[] = []
+
+  functionExtensionsWithUiHandle.forEach((extension) => {
+    const uiHandle = extension.configuration.ui?.handle!
```

Comment:
Nit: We know that `ui.handle` must be in these extensions, so I think we want:

```suggestion
    const uiHandle = extension.configuration.ui!.handle!
```